src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java

```diff
@@ -0,0 +1,119 @@
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+ 
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import java.util.List;
+ 
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.ConfigureMetadataPage;
+import org.openmrs.contrib.qaframework.page.OpenConceptLabPage;
+import org.openmrs.contrib.qaframework.page.OpenConceptLabSuccessPage;
+import org.openmrs.contrib.qaframework.page.SubscriptionPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+ 
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+ 
+public class OpenConceptLabSteps extends Steps  {
+    private OpenConceptLabPage openconceptlabpage;
+    private ConfigureMetadataPage configureMetadataPage;
+    private static final String releasedDictionaryUrl = "https://api.staging.openconceptlab.org/users/openmrs/collections/TD-ecly/65725685/";
+    private static final String tokenUrl = "bd022mad6d3df24f5c42ewewa94b53a23edf6eee7r";
+    private static final String newDictionaryUrl = "http://api.openconceptlab.com/orgs/CIEL/sources/CIEL/";
+    private static final String newTokenUrl = "bd022mad6d3df24f5c42ewewa94b53a23edf6fff7r";
+    private OpenConceptLabSuccessPage openConceptLabSuccessPage;
+    private OpenConceptLabPage openConceptLabPage;
+    private SubscriptionPage subscriptionPage;
+    
+    @Before(RunTest.HOOK.SELENIUM_OPEN_CONCEPT_LAB)
+    public void visitHomePage() {
+        initiateWithLogin();    
+    }
+    
+    @After(RunTest.HOOK.SELENIUM_OPEN_CONCEPT_LAB)
+    public void destroy(){
+        quit();
+    }
+    
+    @Given("a user clicks on configure Metadata link from home page")
+    public void launchMetadataDashboard() {
+    	configureMetadataPage = (ConfigureMetadataPage) homePage.goToConfigureMetadata();
+    }
+    
+    @When("the system loads the configure metadata page")
+    public void systemLoadsConfigureMetadataPage() {
+    	assertPage(configureMetadataPage.waitForPage());
+    }
+ 
+    @And("a user clicks Manage OCL link from configure metadata page")
+    public void loadOpenConceptLabPage() {
+        openconceptlabpage = (OpenConceptLabPage) configureMetadataPage.goToOpenConceptLabPage();
+    }
+    
+    @Then("the system loads Open Concept Lab page")
+    public void systemLoadsOpenConceptPage() {
+        assertPage(openconceptlabpage.waitForPage());
+    }
+    
+   @When ("a user clicks on Setup subscription button")
+    public void loadSubscriptionPage() {
+      subscriptionPage = openconceptlabpage.clickOnsetupSubscriptionButton();
```

Comment:
@jwnasambu have you taken a look at the logs at? https://github.com/openmrs/openmrs-contrib-qaframework/runs/4437237295?check_suite_focus=true#step:7:11159

is clickOnSetupSubscriptionButton() existing any where within [OpenConceptLabPage](https://github.com/openmrs/openmrs-contrib-qaframework/blob/37bbc55d70c17b18051dedbc2d816a7364564154/src/test/java/org/openmrs/contrib/qaframework/page/OpenConceptLabPage.java#L1-L82)?

Reply:
@sharif thanks for the review actually as @kdaud pointed out the error was originating from a method  which doest exist.